proto/one/operations/sample/analyte.proto

```diff
@@ -9,13 +9,13 @@ message Analyte {
     string id = 1;
     
     // Used to check twin authorization
-    google.protobuf.StringValue authTwinRefId = 2;
+    string authTwinRefId = 2;
 
     // Parameter Id
-    google.protobuf.StringValue parameterId = 3;
+    string parameterId = 3;
```

Comment:
I think this was correct before since it is nullable. Same for unitTypeId below.

Reply:
Sorry, I just checked the confluence documentation. It looks like you are correct and parameterId and unitTypeId are not nullable.